routes/models/proposal.js

```diff
@@ -0,0 +1,132 @@
+var md5= require('md5') ;
+var DB = require('../config.js') ;
+var random = require('../../libs/random.js') ;
+var Boom =  require('boom');
+
+module.exports.add = function(req,res) {
+    var proposal = req.payload ;
+    proposal.created =  new Date();
+    proposal.is_active = 1;
+    DB.conn.queryAsync("INSERT INTO "+DB.proposals +" SET ?" , proposal ).then(function(result)
+    {
+        res({"statusCode":200,"message":"New proposal created successfully"}) ;
+    });
+}
+
+module.exports.proposals = function(req,res) {
+
+    var total_count = 0 ;
+
+    if(!req.query.page)
+    {
+        res(Boom.badRequest('Please provide the page number'));
+    }
+
+    if(req.query.page <= 0)
+    {
+        res(Boom.badRequest('Invalid page number.'));
+    }
+
+    var prStatus = req.params.status ;
+    var perpage =  1;
+    var page = req.query.page ;
+    var search = false   ;
+    if(req.query.s) {
+      search = req.query.s ;
+    }
+
+    //if(req.query.perpage){ perpage = req.query.perpage; }
+    if(req.query.page){ page = req.query.page ; }
+    var skip = perpage*(page-1) ;
+    var qry = "SELECT Proposals.* , User.id , User.first_name , Client.id,  Client.name FROM "+DB.proposals +" As Proposals INNER JOIN "+DB.users +" As User ON Proposals.assigned_to = User.id" + " INNER JOIN "+DB.clients + " As Client ON Proposals.client_id = Client.id where Proposals.is_active = "+prStatus ;
+
+    if(search) {
+        qry += " AND (Proposals.title like '%"+search+"%'";
+        qry += " OR User.first_name like '%"+search+"%'";
+        qry += " OR User.last_name like '%"+search+"%'";
+        qry += " OR Client.name like '%"+search+"%') ";
+    }
+
+    qry += ' limit '+perpage+' offset '+skip ;
+    if(req.query.assigned_to) { assigned_to = req.query.assigned_to ; }
+    var options = {sql: qry , nestTables: true } ;
+
+    var total_qry = "select count(*) as count from "+DB.proposals ;
```

Comment:
@tusharncuelogic @sandeepp-cuelogic @gauravc-cuelogic @pritig-cuelogic why you are not selecting only active proposals? In above query there is condition for is_active column?